bucket/tortoisehg.json

```diff
@@ -0,0 +1,30 @@
+{
+    "version": "5.7.1",
+    "description": "TortoiseHg. A set of graphical tools and a shell extension for the Mercurial distributed revision control system.",
+    "homepage": "https://tortoisehg.bitbucket.io/",
+    "license": "GNU General Public License",
+    "architecture": {
+        "64bit": {
+            "url": "https://www.mercurial-scm.org/release/tortoisehg/windows/tortoisehg-5.7.1-x64.msi",
+            "hash": "0221163b190641e422144c55a1088f1cf1a104d47c81175ec1597b44bfdb23dd"
+        },
+        "32bit": {
+            "url": "https://www.mercurial-scm.org/release/tortoisehg/windows/tortoisehg-5.7.1-x86.msi",
+            "hash": "02192542a79b8bc4b312f087cd71242ae6a04fe6c47dcf3d6f78e2cd29e725a8"
+        }
+    },
+    "extract_dir": "PFiles\\TortoiseHg",
+    "bin": [
+        "thgw.exe"
+    ],
+    "checkver": {
+        "url": "https://www.mercurial-scm.org/release/tortoisehg/windows/",
+        "re": "tortoisehg-([\\d\\.]+)-(x64|x86).msi"
+    },
+    "shortcuts": [
+        [
+            "thgw.exe",
+            "TortoiseHg Workbench"
+        ]
+    ]
```

Comment:
add autoupdate

Reply:
> Add reg files to manually register all context menus
Is there a way for scoop to differentiate among different languages in the context menu?  The context menu comes in 24 different language mutations.

Everything else is clear and I'll fix it and send patches for that.

Reply:
One more question.  I would like to detect if `hg.exe` is already in path.  If yes, I would like to put the `env` tortoisehg path after the current `hg.exe` path.  Is that possible?

Reply:
https://github.com/lukesampson/scoop-extras/pull/5987/commits/f4a29088629e53a3f4e11ee0526d35485c61fce8